app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -220,7 +223,14 @@
         %dd.app-check-your-answers__change
           = link_to t('buttons.change'), application_details_school_job_path(anchor: 'expires_on'), 'aria-label': t('jobs.aria_labels.change_application_deadline'), class: 'govuk-link'
 
-    %h2.govuk-heading-m Now submit your job
-    %p= t('jobs.confirmation_summary')
-
-    = link_to t('jobs.submit'), school_job_publish_path(@vacancy.id), method: :post, class: "govuk-button"
+    %h2.govuk-heading-m 
+      - if @vacancy.publish_on.today?
+        = t('jobs.publish_for') + @vacancy.job_title
+      - else 
+        = t('jobs.publish_for_future') + @vacancy.publish_on.to_s
+    %p= t('jobs.confirmation_summary').html_safe
```

Comment:
Instead of invoking the .html_safe method, I would call the translation field `confirmation_summary_html` so that rails does this magic.